test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -682,8 +678,14 @@ def _op_is_unmatchable(op):
                     _op_in_base_sets_of_related_ops(base_op),
                     f"{base_op} not in sets of related ops")
             else:
-                raise AssertionError(
-                    f"handing for {qhandler_cls} not implemented")
+                # torch.sum does not have quantized equivalents
+                if base_op == torch.sum:
+                    continue
+                # didn't match explicit quantize handler class, we can check if the
+                # operator is in the related op set directly
+                if not _op_in_base_sets_of_related_ops(base_op):
```

Comment:
@vkuzo please take a look at this change

Reply:
looks like this is now testing a smaller set of ops, could we keep the set of ops under test the same?  This would mean querying the structure you added (`get_native_backend_config_dict`) and taking the union of ops in there and ops in quantize handlers.

Reply:
not sure I follow, are you talking `get_default_quant_patterns` not returning the patterns in `get_native_backend_config_dict`?

Reply:
oh, my bad, I didn't realize that you added this in `get_all_quant_patterns`, sorry. Look good!

Reply:
oh I corrected it later after your comment, although some how the new patterns does appear as well, probably because we are modifying the patterns dictionary inplace..

Reply:
I'll change get_quant_patterns to return a copy, instead of the original dictionary